scripts/tasks/jest.ts

```diff
@@ -1,22 +1,44 @@
-import { jestTask, argv } from 'just-scripts';
+import { jestTask, argv, JestTaskOptions } from 'just-scripts';
 import * as path from 'path';
 
-const commonArgs = () => {
+/**
+ * Partial support of native jest CLI arguments https://jestjs.io/docs/en/cli
+ * > Why partial support only? just jestTask limits support to only those specified as return value of this function
```

Comment:
This actually isn't quite true, jestTask allows passing any extra args as `JestTaskOptions._`.

Reply:
that's unfortunately not true.

`_` is mapped to positional arguments (yargs behaviour), 

so `just jest --watch` -> parses to `{_: [],}
where `just jest foobar -> parses to `{_: ['foobar']}

Reply:
Got it. I think if you want to directly run jest with particular options, an easier way is `yarn jest --whatever --options` (for example you can already do `--clearCache` that way).

Reply:
yeah, but in current setup ( devDeps all over the place) the result might differ - various versions of jest.
 
> This will be mitigated when we adopt single version policy 💪, so we can invoke tools without wrappers or have good wrappers that support `--help` and other related stuff